controllers/enable_mesh.go

```diff
@@ -79,8 +91,8 @@ func newServiceMeshMember(ns *v1.Namespace) *maistrav1.ServiceMeshMember {
 		Spec: maistrav1.ServiceMeshMemberSpec{
 			// TODO should we make it configurable?
 			ControlPlaneRef: maistrav1.ServiceMeshControlPlaneRef{
-				Name:      "basic",
-				Namespace: MeshNamespace,
+				Name:      controlPlaneName,
+				Namespace: meshNamespace,
```

Comment:
We can remove `MeshNamespace` entirely now

Reply:
It's still called in `findIstioIngress()` didnt want to modify the logic of this function as it was outside of the scope of the issue. https://github.com/cam-garrison/odh-project-controller/blob/4fabb125fff2f6f05e84bc502e874f700c725b13/controllers/enable_mesh.go#L118

Should this be handled in a separate PR/Issue? Or here as well.

Reply:
It is in scope of this issue I believe. We change ns of istio-system so the lookup should also take it into consideration. Otherwise, the logic is broken :)